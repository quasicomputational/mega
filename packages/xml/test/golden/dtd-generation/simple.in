<entity-set>
  <entity name="foo">&amp;hello!</entity>
  <entity name="bar">&lt;markup</entity>
  <entity name="baz">newline&#xA;separated</entity>
  <entity name="qux">&#x2012;</entity>
</entity-set>
