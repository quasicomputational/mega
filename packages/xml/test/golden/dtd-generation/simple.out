<!ENTITY foo "&#x26;#x26;&#x26;#x68;&#x26;#x65;&#x26;#x6c;&#x26;#x6c;&#x26;#x6f;&#x26;#x21;">
<!ENTITY bar "&#x26;#x3c;&#x26;#x6d;&#x26;#x61;&#x26;#x72;&#x26;#x6b;&#x26;#x75;&#x26;#x70;">
<!ENTITY baz "&#x26;#x6e;&#x26;#x65;&#x26;#x77;&#x26;#x6c;&#x26;#x69;&#x26;#x6e;&#x26;#x65;&#x26;#xa;&#x26;#x73;&#x26;#x65;&#x26;#x70;&#x26;#x61;&#x26;#x72;&#x26;#x61;&#x26;#x74;&#x26;#x65;&#x26;#x64;">
<!ENTITY qux "&#x26;#x2012;">
